cogs/util/bot.py

```diff
@@ -76,13 +76,17 @@ def _check_table_exists(self, tablename):
         allowed = channel_ids.get('allowed_channels', None)
         blocked = channel_ids.get('blocked_channels', [])
 
+        # WTF is this?!?!?! This is horrible code!
+        # Not that I'm helping though.
+        # -hanss314
+        
         if allowed is not None:
             if channel.id not in allowed:
```

Comment:
can't you use checks for this?

Reply:
That would mean a bit of refactoring which I was too lazy to do.